test/v1-large/condition_test.py

```diff
@@ -0,0 +1,41 @@
+from kfp import components
```

Comment:
This `test/v1-large` folder contains 14 core sample files updated to run as proper large tests executed by pytest. These replace the existing e2e test workflow (wrt to the SDK) and sample test workflow, which collectively covered less ground despite being far more complex (and redundant).